src/Symfony/Component/Console/EventListener/ExceptionListener.php

```diff
@@ -11,16 +11,20 @@
 
 namespace Symfony\Component\Console\EventListener;
 
+@trigger_error(sprintf('The "%s" class is deprecated since version 3.3 and will be removed in 4.0. Use the ErrorListener class instead.', ExceptionListener::class), E_USER_DEPRECATED);
+
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Console\Event\ConsoleEvent;
 use Symfony\Component\Console\ConsoleEvents;
-use Symfony\Component\Console\Event\ConsoleErrorEvent;
+use Symfony\Component\Console\Event\ConsoleExceptionEvent;
 use Symfony\Component\Console\Event\ConsoleTerminateEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
 /**
  * @author James Halsall <james.t.halsall@googlemail.com>
  * @author Robin Chalas <robin.chalas@gmail.com>
+ *
+ * @deprecated since version 3.3, to be removed in 4.0. Use ErrorListener instead.
```

Comment:
This listener has been added in 3.3 (https://github.com/symfony/symfony/pull/21003), can be renamed directly :)